sw/device/silicon_creator/mask_rom/mask_rom.ld

```diff
@@ -86,6 +86,20 @@ SECTIONS {
     *(.text)
     *(.text.*)
 
+
+    /* Ensure section end is word-aligned. */
+    . = ALIGN(4);
+  } > rom
+
+  /**
+   * Shutdown text section, containing shutdown function(s).
+   *
+   * This must be the last executable section in the ROM.
```

Comment:
Presumably this is so that executing past the end of the function doesn't run into anything else, right?

Maybe we want to `FILL` the section with `unimp`s as well.